README.md

```diff
@@ -204,25 +253,146 @@ This endpoint returns a list of all existing criteria. It will return an empty a
+| category_id         | Integer  | *Optional*. The id of the category to which the subcategory is related. This cannot be changed, and will return a 400 if it differs from the existing value. |
+| state_code          | String   | *Optional*. The state to which the subcategory is related. This cannot be changed, and will return a 400 if it differs from the existing value. |
+| text                | String   | *Optional*. |
+| url                 | String   | *Optional*. |
+| active              | Boolean  | *Optional*. Passing in `false` will create a deactivated link. Links cannot be reactivated once they have been deactivated. |
+
+### Scores
+
+A score represents whether a state meets a certain criteria.
+
+| Name            | Type    | Notes                   |
+|-----------------|---------|-------------------------|
+| id              | Integer | Primary key             |
+| state_code      | String  | Foreign key             |
+| criterion_id    | Integer | Foreign key             |
+| meets_criterion | Boolean |                         |
+
+#### POST /scores
+
+This endpoint creates a new score. Note that scores CANNOT be updated, only overwritten. To overwrite
+a score, create a new score for the same state and criterion.
+
+This endpoint requires [authentication](#Authentication). It accepts a JSON body with the following format:
+
+| Name            | Type    | Notes                                                     |
+|-----------------|---------|-----------------------------------------------------------|
+| state_code      | String  | *Required*. The state code to which the score is related. |
+| criterion_id    | Integer | *Required*. The criterion to which the score is related.  |
+| meets_criterion | Boolean | *Required*. Whether the state meets the criterion.        |
+
 ### Grades
 
 A **state grade** represents the overall grade assigned to a state based on their survivor wealth policies.
 
 |    Name    |  Type   |          Notes          |
 |------------|---------|-------------------------|
 | id         | Integer | Primary key             |
-| state_code | String  |                         |
+| state_code | String  | Foreign key             |
 | grade      | Integer | One of (-1, 0, 1, 2, 3) |
 
 A **state category grade** represents the grade assigned to a state based on a specific category.
 
 |    Name     |  Type   |          Notes          |
 |-------------|---------|-------------------------|
 | id          | Integer | Primary key             |
-| state_code  | String  |                         |
-| category_id | Integer |                         |
+| state_code  | String  | Foreign key             |
+| category_id | Integer | Foreign key             |
 | grade       | Integer | One of (-1, 0, 1, 2, 3) |
 
 #### GET /grades/{code}
 
 Returns the state's overall grade, and its grades for each category. If no state with that code exists, it will return a 404 response code.
+
+#### POST /grades/{code} (UPCOMING)
```

Comment:
Let me know what you think of this endpoint and the one below as ideas for how we could create and update the state grades/state category grades!

Reply:
Alternatively, we could do `POST /grades` and `POST /category-grades` or something and put the state code and category id in the post body.